neutrino-custom/lint.js

```diff
@@ -1,92 +1,77 @@
 'use strict';
+const eslint = require('neutrino-middleware-eslint');
 
-const merge = require('deepmerge');
```

Comment:
deepmerge is now unused be Treeherder itself, so can be removed from package.json